runtimes/pythonrt/ak_runner.py

```diff
@@ -379,5 +382,14 @@ def module_entries(mod):
     event = message.get('event')
     event = {} if event is None else event
 
+    # Inject HTTP body
+    # TODO (ENG-624) change this once we support callbacks to autokitteh
+    body = event.get('data', {}).get('body')
```

Comment:
oh wow this is very specific in a very generic place

Reply:
Yes, I'll re-do it once we have stabler event API to Python.